main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -509,6 +510,43 @@ object CompletionProvider {
     }
   }
 
+  /** 
+    * Returns a list of completion items based on case keyword in match expressions
+    */
+  private def getCaseCompletions()(implicit context: Context, index: Index, root: TypedAst.Root): Iterable[CompletionItem] = {
+    if (root == null) {
+      return Nil
+    }
+    
+    //TODO Do not match when declaring enums.
```

Comment:
We don't have TODOs in code (preferring to keep them in tickets on GitHub or to fix them)